### PR TITLE
feat: Implement custom user agent

### PR DESCRIPTION
**Pull request by marwin-roth**
Custom User agent to query logs of customers actually using the python SDK

### CODE REVIEW

File: emnify/constants.py [hunk 1]
```diff
@@ -39,13 +40,15 @@ class RequestDefaultHeadersKeys(ExtendedEnum):
...
     XEmnOriginApp = 'x-emn-origin-app'
     XEmnOriginAppVersion = 'x-emn-origin-app-version'
+    USER_AGENT = 'User-Agent'
 
 
 class RequestDefaultHeadersValues(ExtendedEnum):
     APPLICATION_JSON = 'application/json'
     BEARER_TOKEN = 'Bearer {}'
     PYTHONSDK_VERSION = EMNIFY_PACKAGE_VERSION
     PYTHONSDK = 'PythonSDK'
```
Comment (StephenKing) [inline emnify/constants.py:50]:
```suggestion
    PYTHONSDK = 'emnify PythonSDK'
```

Comment (StephenKing) [reply emnify/constants.py:50]: wondering what a good pattern would be.. assuming we have multiple SDKs following that schema, I guess a leading `emnifySDK` and only afterwards (somewhere) the language would make sense to filter more easily.

File: emnify/constants.py [hunk 2]
```diff
@@ -39,13 +40,15 @@ class RequestDefaultHeadersKeys(ExtendedEnum):
...
     XEmnOriginAppVersion = 'x-emn-origin-app-version'
+    USER_AGENT = 'User-Agent'
 
 
 class RequestDefaultHeadersValues(ExtendedEnum):
     APPLICATION_JSON = 'application/json'
     BEARER_TOKEN = 'Bearer {}'
     PYTHONSDK_VERSION = EMNIFY_PACKAGE_VERSION
-    PYTHONSDK = 'PythonSDK'
+    PYTHONSDK = 'emnify PythonSDK'
```
Comment (Th3Un1q3) [inline emnify/constants.py:50]: No spaces please. `emnifyPythonSDK`

File: emnify/constants.py [hunk 3]
```diff
@@ -39,13 +40,15 @@ class RequestDefaultHeadersKeys(ExtendedEnum):
...
+    USER_AGENT = PYTHONSDK + '/' + PYTHONSDK_VERSION + ' ('+ platform.system() + ', ' + platform.release() + ')'
```
Comment (Th3Un1q3) [inline emnify/constants.py:51]: Consider including python version as well.

**Review by Th3Un1q3 — CHANGES_REQUESTED** [review]

File: emnify/constants.py (hunk 3 above)
Comment (Th3Un1q3) [reply emnify/constants.py:51]: Include requests version too

Comment (Th3Un1q3) [reply emnify/constants.py:51]: Some example I can see:

`emnifyPythonSDK/1.0.0 requests/2.26.0 (Python 3.9; +https://github.com/emnify/emnify-python-sdk)`

Comment (marwin-roth) [reply emnify/constants.py]: This would be the new User agent`emnifyPythonSDK/0.3.11 python-requests/2.29.0 (Python 3.13.0, Linux, 6.6.32-linuxkit)`

**Review by Th3Un1q3 — APPROVED** [review]

File: emnify/constants.py (hunk 1 above)
Comment (Th3Un1q3) [reply emnify/constants.py]: You can find current example [here](https://github.com/emnify/emnify-sdk-python/pull/253/files#diff-1ec6ad7a01cbbfe90ad8531ae82907860658905459cd44a66f5405b4e648b661R10), it aligns with standaard, and includes variety of relevant info.

**Pull request merged**